src/components/editor/feature-action-panel.js

```diff
@@ -58,9 +60,13 @@ export class FeatureActionPanel extends PureComponent {
       datasets,
       position,
       layers,
+      currentFilter,
+      onToggleLayer,
       onDeleteFeature
     } = this.props;
 
+    const {layerId = []} = (currentFilter || {});
+
     return (
       <StyledActionsLayer className={className} position={position}>
```

Comment:
instead of passing this to styled-component. this should just be in component `style={{top: x, left: y}}` handled by React. It is not recommended to pass props that constantly change to styled-component, because it needs to inject to css whenever prop changes.